journal/modernized/38-chapter-35.adoc

```diff
@@ -332,6 +332,7 @@ but living and substantial truth.
 +++[+++The following is a letter of Daniel Wheeler written about this time to John Wilbur:^
 footnote:[This letter was not taken from the Journal of the Life and Labour
 of John Wilbur and included in this memoir by The Friends Library.]]
+//It says "This letter was NOT taken..." This was circled in the book....is it right the way it is? Is it stating it wasn't taken from John Wibur's journal? idk. 
```

Comment:
Yup, you're right, this is a mistake jason made, because he added this footnote. Remove the `not`.